### PR TITLE
Update Android Components version

### PR DESCRIPTION
**Pull request by MickeyMoz**

### CODE REVIEW

**Comment by csadilek** [comment]
@liuche @kglazko looks like detekt is failing on master due to missing license files for downloads:

```
[task 2020-08-19T20:03:34.060Z] 	AbsentOrWrongFileLicense - [DownloadController.kt] at /builds/worker/checkouts/src/app/src/main/java/org/mozilla/fenix/library/downloads/DownloadController.kt:1:1
[task 2020-08-19T20:03:34.060Z] 	AbsentOrWrongFileLicense - [DownloadAdapter.kt] at /builds/worker/checkouts/src/app/src/main/java/org/mozilla/fenix/library/downloads/DownloadAdapter.kt:1:1
[task 2020-08-19T20:03:34.060Z] 	AbsentOrWrongFileLicense - [DownloadView.kt] at /builds/worker/checkouts/src/app/src/main/java/org/mozilla/fenix/library/downloads/DownloadView.kt:1:1
[task 2020-08-19T20:03:34.061Z] style - 5min debt
[task 2020-08-19T20:03:34.061Z] 	MaxLineLength - [DownloadItem] at /builds/worker/checkouts/src/app/src/main/java/org/mozilla/fenix/library/downloads/DownloadFragmentStore.kt:19:1
```

**Comment by csadilek** [comment]
Likely caused by the newlines at the beginning of the files: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/library/downloads/DownloadController.kt#L1

**Comment by kglazko** [comment]
Thank you, I will fix this right away. For some reason, Detekt was passing locally and the CI passed completely on the PR :(

File: buildSrc/src/main/java/AndroidComponents.kt [hunk 1]
```diff
@@ -3,5 +3,5 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object AndroidComponents {
-    const val VERSION = "54.0.20200818130156"
+    const val VERSION = "56.0.20200819190136"
```
Comment (jonalmeida) [inline buildSrc/src/main/java/AndroidComponents.kt:6]: How did we skip 55? 🤔

Comment (liuche) [reply buildSrc/src/main/java/AndroidComponents.kt:6]: Is there a bug in the script? The PR merged yesterday bumped to a 54 version. https://github.com/mozilla-mobile/fenix/commit/5f5caa4720c1f18637ba12946204e8f162da7b83

Or maybe whoever did the previous release forgot to bump the version number to 55 after they did the 54 cut?

Comment (jonalmeida) [reply buildSrc/src/main/java/AndroidComponents.kt:6]: Ah, it was an error in the release. We'll have to document that 55 was skipped in error and maybe write a release script check to ensure the increments are exactly i++.

**Comment by csadilek** [comment]
> Or maybe whoever did the previous AC release forgot to bump the version number to 55 after they did the 54 cut?

Yes, this is what happened.

**Comment by csadilek** [comment]
@jonalmeida all releases went out fine, including 55. It's just whoever released 54 forgot to increment the version right away. So the Nightlies had the wrong version (54*) for an extra week.

**Pull request merged**